### DISCUSSION TITLE
URL::signedRoute Starts with api. after changing the queue driver to google cloudtask

### DISCUSSION THREAD

**Discussion by dumithsalinda** [stackkit/laravel-google-cloud-tasks-queue#150] [General] [answered]
I'm using google app engine for the project. recently changed the queue driver to google cloud task queue. After changing the queue driver, The signed route on my email return the domain starting from **api.Domain.com**. Which was earlier **domain.com**

The following code used in the email.

```
    public function toMail($notifiable)
    {
        $url = URL::signedRoute('activate', [
            'email' => $notifiable->email,
        ]);

        return (new MailMessage)
            ->subject('Activate Your Account')
            ->markdown('mails.auth.activate', [
                'url' => $url,
                'email' => $notifiable->email,
            ]);
      
    }
```

**Comment by marickvantuil** [comment]
I tried to reproduce this, but I can't. Do you have a reproducible example?

**Reply by marickvantuil** [reply]
The package doesn't do anything with signed routes, so I'm not entirely sure this package is the culprit.

**Reply by dumithsalinda** [reply, marked as answer]
Not sure what causes the issue. but I've fixed the issue by adding ` URL::forceRootUrl(config('app.url'));` in the Service Provider. 

Thank you.
